### PR TITLE
ADD deprecatedFeatures counters

### PR DESCRIPTION
**Pull request by fgalan**

### CODE REVIEW

File: test/functionalTest/cases/0000_statistics_operation/statistics_with_counters.test [hunk 1]
```diff
@@ -249,7 +249,7 @@ orionCurl --url /v1/updateContext                -X POST --payload "$payload"  >
 
 # Get statistics
 
-orionCurl --url /statistics?options
+orionCurl --url /statistics
```
Comment (fgalan) [inline test/functionalTest/cases/0000_statistics_operation/statistics_with_counters.test:252]: Not directly related with the purpose of this PR, this is just a typo fix.

NTC

**Comment by fgalan** [comment]
@fisuda this PR does a slight modification to a couple of .md documents. It would be great if you could sync the Japanese translation, please. Thanks!

File: test/functionalTest/cases/3004_v2_reg_create/v2_reg_create.test [hunk 2]
```diff
@@ -138,10 +138,13 @@ HTTP/1.1 200 OK
 Date: REGEX(.*)
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Content-Type: application/json
-Content-Length: 206
+Content-Length: 250
 
 {
     "counters": {
+        "deprecatedFeatures": {
```
Comment (mapedraza) [inline test/functionalTest/cases/3004_v2_reg_create/v2_reg_create.test:145]:
```suggestion
        "deprecatedRequests": {
```

Maybe the name of this field should be `deprecatedRequests` in order to align with the rest of the field?

Comment (fgalan) [reply test/functionalTest/cases/3004_v2_reg_create/v2_reg_create.test:145]: Discussed at Teams.

Although most of the counters are xxxxRequests, in this case, it seems better `deprecatedFeatures`.

NTC

**Review by mapedraza — APPROVED** [review]
LGTM

**Comment by fisuda** [comment]
I sent the PR #4483. Thanks.

**Pull request merged**